src/main/java/carpetaddonsnotfound/mixins/SpawnHelper_disableMobSpawningInMixin.java

```diff
@@ -0,0 +1,27 @@
+package carpetaddonsnotfound.mixins;
+
+import carpetaddonsnotfound.CarpetAddonsNotFoundSettings;
+import net.minecraft.server.world.ServerWorld;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.SpawnHelper;
+import net.minecraft.world.World;
+import net.minecraft.world.chunk.Chunk;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+@Mixin(SpawnHelper.class)
+public abstract class SpawnHelper_disableMobSpawningInMixin {
+  @Inject(method = "isAcceptableSpawnPosition", at = @At("HEAD"), cancellable = true)
+  private static void isAcceptableSpawnPosition(final ServerWorld world, final Chunk chunk, final BlockPos.Mutable pos,
```

Comment:
Guessing the `final` modifiers were added automatically by the IDE? Didn't realise that would be done (I'll need to modify the IDE code styling rules since they're not really needed.

Did you run the "Reformat Code" option with "optimize imports" and "cleanup code" on this file?

Reply:
> Did you run the "Reformat Code" option with "optimize imports" and "cleanup code" on this file?

Yes